mgmtsystem_action/reports/mgmtsystem_action_report.py

```diff
@@ -34,7 +34,6 @@ class MgmtsystemtActionReport(models.Model):
     stage_id = fields.Many2one("mgmtsystem.action.stage", "Stage", readonly=True)
     system_id = fields.Many2one("mgmtsystem.system", "System", readonly=True)
 
-    @api.model_cr
     def init(self):
         """Display a pivot view of action."""
         tools.drop_view_if_exists(self._cr, "mgmtsystem_action_report")
```

Comment:
Can you use the same approach than new views for favoring inheritance?

Example: https://github.com/odoo/odoo/blob/c32126da5ac55a03637d82cf520b8655be8546e6/addons/sale/report/sale_report.py#L147